app/services/meta-tags.ts

```diff
@@ -0,0 +1,184 @@
+import { service } from '@ember-decorators/service';
+import Service from '@ember/service';
+import config from 'ember-get-config';
+import pathJoin from 'ember-osf-web/utils/path-join';
+
+export type NameAttr = 'name' | 'property';
+
+export type Content = string | number | null | undefined;
+
+export type DataContent = Content | Content[];
+
+/* eslint-disable camelcase */
+export interface MetaTagsData {
+    title: DataContent;
+    type?: DataContent;
+    description?: DataContent;
+    url?: DataContent;
+    doi?: DataContent;
+    identifier?: DataContent;
+    published_date?: DataContent;
+    modified_date?: DataContent;
+    license?: DataContent;
+    language?: DataContent;
+    image?: DataContent;
+    image_type?: DataContent;
+    image_width?: DataContent;
+    image_height?: DataContent;
+    image_alt?: DataContent;
+    site_name?: DataContent;
+    institution?: DataContent;
+    twitter_site?: DataContent;
+    twitter_creator?: DataContent;
+}
+/* eslint-enable camelcase */
+
+export interface MetaTagsDefs {
+    [s: string]: DataContent;
+}
+
+export interface NameMetaTagAttrs {
+    name: string; // eslint-disable-line no-restricted-globals
```

Comment:
What's the restricted global here?

Reply:
ohh, we can't use anything that's a property on `window`? I guess that makes sense, but could be annoying.